.buildkite/scripts/serverless/create_deploy_tag/info_sections/build_info.ts

```diff
@@ -80,7 +80,7 @@ export async function getQAFBuildContainingCommit(
   // Find the first build that contains this commit
   const build = qafBuilds
     // Only search across scheduled builds, triggered builds might run with different commits
-    .filter((e) => e.source === 'schedule')
+    .filter((e) => e.env.PRE_RELEASE_CHECK?.match(/(1|true)/i) || e.source === 'schedule')
```

Comment:
Do you think we still need the `e.source === 'schedule'` bit or could this be dropped?

Reply:
Yes. I kept it for backwards compatibility, but I didn't consider that the next release will be a week from now, so we won't really need to consider any from the old, un-tagged variants.